jquery.coverflow.js

```diff
@@ -119,8 +119,9 @@
 			// Swipe
 			if ($.isFunction(that.element.swipe)) {
 				that.element.swipe({
+					allowPageScroll: "vertical",

```

Comment:
allowPageScroll: "vertical" allows for vertical swipes on the element to scroll up and down the page. allowPageScroll: "auto" would also work since we haven't defined actions for up and down swipe, but I feel like vertical is more easily understood.